cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_nodegroup_test.go

```diff
@@ -437,8 +438,8 @@ func TestNodeGroupDecreaseTargetSize(t *testing.T) {
 			t.Fatalf("failed to add new machine: %v", err)
 		}
 
-		if err := ng.DecreaseTargetSize(tc.delta); err != nil {
-			t.Fatalf("unexpected error: %v", err)
+		if err := ng.DecreaseTargetSize(tc.delta); (err != nil) != tc.expectedError {
```

Comment:
It's not clear why the existing assertion shouldn't hold true; perhaps it's just the test inputs that need changing.